core/src/main/scala/kafka/log/Log.scala

```diff
@@ -253,6 +253,15 @@ class Log(@volatile var dir: File,
   }
 
   private def removeTempFilesAndCollectSwapFiles(): Set[File] = {
+
+    def deleteIndicesIfExist(baseFile: File, swapFile: File, fileType: String): Unit = {
+      info(s"Found $fileType file ${swapFile.getAbsolutePath} from interrupted swap operation. Deleting index files (if they exist).")
```

Comment:
I thought it's a good idea to log whenever we delete files.

Reply:
@junrao do you think we should remove fileType from here too?

Reply:
@ijuma : Ok, it seems many existing logging has the fileType already. So, we can just keep the file type as what you have now.